roles/heat/templates/etc/heat/heat.conf

```diff
@@ -37,7 +37,11 @@ instance_connection_is_secure=1
 region_name_for_services=RegionOne
 
 rpc_thread_pool_size=64
+{% if ceilometer.enabled|default('True')|bool -%}
```

Comment:
The default for the ceilometer role is enabled: False, so we should follow suite here, and default to disabled.